docs/imagen-base.md

```diff
@@ -0,0 +1,29 @@
+# Imagen base del contenedor
+
+## Criterios de búsqueda de una imagen base:
+
+Para poder completar el [M1](https://github.com/pablo1mc315/iv_pablomc/milestone/3), debemos crear un contenedor donde se ejecuten nuestros tests. Para ello, lo primero es elegir una imagen base sobre la que construir nuestro contenedor.
+
+Siguiendo las buenas prácticas, usaremos una imagen base de Docker oficial o de un publicador verificado, las cuales podemos encontrar en [dockerhub](https://hub.docker.com/search?q=&type=image&image_filter=official%2Cstore). Para elegir una para nuestra aplicación, se establecen los siguientes criterios:
+
+- El tamaño de la imagen es muy importante, puesto que cuanto menos ocupe la misma, más se optimizará el rendimiento y menos tiempo tardarán los tests en ejecutarse.
+- Debemos tener en cuenta que vamos a usar nuestro contenedor para testear nuestra aplicación, por lo que cualquier funcionalidad extra lo único que hace es añadir deuda técnica al proyecto, cosa que no queremos. 
+- Debe tener soporte actualmente y estar en continuo mantenimiento, puesto que no queremos que nuestra aplicación quede obsoleta.
+
+## Imágenes base posibles:
+
+### node:alpine
```

Comment:
¿No hay más imágenes base? ¿Como las que os expliqué en clase?

Reply:
Sé que hay más imágenes, pero como me ha dicho usted otras veces, no creo que deba ponerlas si no cumplen los requisitos como "no tener el mínimo peso posible" o "tener instalaciones innecesarias que no me sirvan a la hora de simplemente ejecutar test y que hagan que el tamaño aumente considerablemente".

Reply:
¿Como sabes que no tiene el mínimo peso? ¿O que las instalaciones son innecesarias? ¿Has puesto eso como requisito? ¿Has listado las que hay y por qué no lo cumplen?

Reply:
¿Has buscado las mejores prácticas en imágenes que incluyan node? ¿Qué es lo que dicen?

Reply:
Según he podido leer [aquí](https://snyk.io/blog/10-best-practices-to-containerize-nodejs-web-applications-with-docker/), la imagen de `node:alpine` no es buena elección porque "muchos escáneres de vulnerabilidades de seguridad no pueden detectar fácilmente artefactos de software o tiempos de ejecución en imágenes de Node en Alpine, lo que es contraproducente para los esfuerzos por proteger las imágenes de su contenedor."

Por tanto la descarto y añado otra como `node:lts-bullseye-slim` que es la que se recomienda en esa misma guía de buenas prácticas.

Reply:
He creado el issue #61 para volver a revisar todo el objetivo en torno a la guía de buenas prácticas y modificar lo que sea necesario.